drivers/clock_control_nrfxlib/Kconfig

```diff
@@ -1,14 +1,24 @@
 #
-# Copyright (c) 2018 Nordic Semiconductor ASA
+# Copyright (c) 2019 Nordic Semiconductor ASA
 #
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-config CLOCK_CONTROL_NRF_LL_NRFXLIB
+#use depends on instead of if else and default y
+config CLOCK_CONTROL_BT_LL_NRFXLIB
 	bool "nRF BLE Controller clock control driver"
-	default y
+	depends on BT_LL_NRFXLIB
 	select CLOCK_CONTROL_NRF_FORCE_ALT
 
+config CLOCK_CONTROL_MPSL_NRFXLIB
+	bool "MPSL nRF clock control driver"
+	depends on MPSL_NRFXLIB
```

Comment:
Done, except the depends on part, because in the drivers/Kconfig we have this: 
`if BT_LL_NRFXLIB
rsource "bt_ll_nrfxlib/Kconfig"
rsource "clock_control/Kconfig"
endif

if MPSL_LIBRARY
rsource "clock_control/Kconfig"
endif`